examples/workflows/unet_evaluation_dict.py

```diff
@@ -56,6 +56,9 @@ def main():
     segs = sorted(glob(os.path.join(tempdir, "seg*.nii.gz")))
     val_files = [{"image": img, "label": seg} for img, seg in zip(images, segs)]
 
+    # model file path
+    model_file = glob("./runs/net_key_metric*")[0]
```

Comment:
If you want to select the best model for evaluation, should we `sort` the model names first?

Reply:
no, only one file is being saved but each time with different metric. I mean earlier it was hardcoded with "./runs/net_key_metric=0.9101.pth" but in my case after running the training script, the model file saved was "./runs/net_key_metric=0.9305.pth". So as it might be different i am taking the regex form as done above.

Reply:
I see, then it looks good to me.
thanks.